### DISCUSSION TITLE
Is it simple to get this running? GPT4ALL

### DISCUSSION THREAD

**Discussion by vdruts** [oobabooga/text-generation-webui#634] [General]
https://github.com/nomic-ai/gpt4all

**Comment by BetaDoggo** [comment]
You should be able to load the [Lora](https://huggingface.co/nomic-ai/gpt4all-lora-epoch-2) for it by following the instructions on the[ wiki](https://github.com/oobabooga/text-generation-webui/wiki/Using-LoRAs).

**Reply by OWKenobi** [reply]
Dumb question: this Lora is based on Llama-7B. Would it also run on top of Llama-30B?
